src/Main.css

```diff
@@ -30,10 +30,17 @@ div.imprint.footer-element {
 }
 
 /**
- * Apply the baseColor to various elements
+ * Apply the toolbarColor to various elements
  */
 button, .ant-collapse-header, .react-geo-titlebar {
-  background-color: var(--baseColor) !important;
+  background-color: var(--toolbarColor) !important;
```

Comment:
Could we leave this by `baseColor` please? This change leads to undesired color palette change effects by existing applicaitons.